src/transformers.jl

```diff
@@ -145,3 +145,29 @@ function apply!(table::T, t::Transform; cols=nothing, kwargs...)::T where T
 
     return table
 end
+
+"""
+    apply!!(A::AbstractArray, ::Transform; dims=1, kwargs...)
+
+Applies the [`Transform`](@ref) to each element of `A` and appends the result to `A`, if
+possible, creating a new array.
```

Comment:
>creating a new array

I'd ideally have liked this to be mutating but it appending doesn't really work like that AFAIK